scripts/jtsync/README.md

```diff
@@ -0,0 +1,57 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+          --ci SERVICE                 Which ci is used (suse or opensuse)
+          --matrix NAME,PROJECT        Set status of a matrix job
+          --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix crowbar-trackupstream,Devel:Cloud:7:Staging
+
+The script never returns anything other than `0` to make sure jenkins run are not interruped
+by the script.
+
+
+## Run the script on local machine
+Install `bundler` before continue:
+
+    cd <path to jtsync directory>
+    bundler install --path=~/.bundle-jtsync
+    bundle exec jtsync.rb
+    ...
+
+## How to test:
+In general there is the `test_jtsync` script to run an manual test run. You need need to create credentials for the trello api.
+Place them in your local `netrc` file:
+
+    machine api.trello.com
+      login <developer token>
```

Comment:
I guess it will be helpful if you can add the link to the trello docu about how you can create the token?